modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -4,15 +4,9 @@
 
 import * as runtime from '../runtime';
 {{#imports.0}}
-import {
-    {{#imports}}
-    {{className}},
-    {{^withoutRuntimeChecks}}
-    {{className}}FromJSON,
-    {{className}}ToJSON,
-    {{/withoutRuntimeChecks}}
-    {{/imports}}
-} from '../models';
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { {{#imports}}{{className}}{{^withoutRuntimeChecks}}, {{className}}FromJSON, {{className}}ToJSON{{/withoutRuntimeChecks}}{{^-last}}, {{/-last}}{{/imports}} } from '../models';
```

Comment:
**question:** hm... I can see how this can still be a problem if the generated file is later reformatted (it's not uncommon to format source files with tools like Prettier).

Would it be more flexible to recommend using https://www.npmjs.com/package/eslint-plugin-unused-imports rule to get rid of unused imports as a post process step?